test/verify/check-openshift

```diff
@@ -406,6 +406,29 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         wait(lambda: "virt-launcher-testvm" in o.execute("oc get pods"))
         wait(lambda: "Running" in o.execute("oc get pods | grep virt-launcher-testvm"))
 
+    def testVirtualMachinesTabPresent(self):
+        self.bootstrapKubevirt()
+
+        b = self.browser
+        self.login_and_go("/kubernetes")
+        b.wait_present("#kubernetes-navigation")
+        self.assertTrue(b.is_present("#vms-menu-link"))
+
+    def testVirtualMachinesListed(self):
+        self.bootstrapKubevirt()
+
+        o = self.openshift
+        o.execute("oc create -f /kubevirt/iscsi-demo-target.yaml")  # disk for test VM
+        o.execute("oc create -f /kubevirt/vm.yaml")  # let's create a VM
+        wait(lambda: "testvm" in o.execute("oc get vm"), msg='Deployment of virtual machine "testvm" failed.')
+
+        b = self.browser
+        self.login_and_go("/kubernetes")
+        b.wait_present("#vms-menu-link")
+        b.click("#vms-menu-link")
+        b.wait_present("tr[data-row-id='vm-testvm']")
+        self.assertTrue(b.is_present("tr[data-row-id='vm-testvm']"))
```

Comment:
Not important, but this is redundant: The wait_present before already asserts the presence.

Could this also check some actual content of the VM, like that it has a reasonable name and node?  Clicking on it should reveal the details pane, which should have some contents asserted.

Are kubevirt nodes the same as "normal" container nodes in openshift? There is no particular implementation of a "nodes" view here, so I assume the nodes link will just lead to the existing pages?

Reply:
Kubevirt is just an extension to k8s, `nodes` are same with or without kubevirt. The idea is to implement click-through in a follow-up patch.